### PR TITLE
Update categoryUser table when update User

### PR DESCRIPTION
**Pull request by david-winder-kaltura**
PLAT4661 #time 1d

### CODE REVIEW

File: alpha/lib/model/categoryKuser.php [hunk 1]
```diff
@@ -43,7 +43,14 @@ public function applyDefaultValues()
 	{
 		$this->setUpdateMethod(UpdateMethodType::MANUAL);
 	}
-	
+
+	public function updateUser($newPuserId = null, $newScreenName = null) {
```
Comment (MosheMaorKaltura) [inline alpha/lib/model/categoryKuser.php]: Rename to updateKuser

Comment (MosheMaorKaltura) [inline alpha/lib/model/categoryKuser.php]: remove the new* from the var names

File: alpha/lib/model/categoryKuser.php [hunk 2]
```diff
@@ -43,7 +43,14 @@ public function applyDefaultValues()
 	{
 		$this->setUpdateMethod(UpdateMethodType::MANUAL);
 	}
-	
+
+	public function updateKuser($PuserId = null, $ScreenName = null) {
```
Comment (MosheMaorKaltura) [inline alpha/lib/model/categoryKuser.php]: Coding conversion  PuserId & ScreenName should be lower cammel case

File: api_v3/services/UserService.php [hunk 3]
```diff
@@ -150,6 +166,11 @@ public function updateAction($userId, KalturaUser $user)
 			}			
 			$dbUser = $user->toUpdatableObject($dbUser);
 			$dbUser->save();
+			if (($user->id || $user->screenName) &&
+				categoryKuserPeer::isCategroyKuserExistsForKuser($dbUser->getId()))
+			{
+				$this->updateCategoryKuser($dbUser->getId(),$user);
```
Comment (tan-tan-kanarek) [inline api_v3/services/UserService.php]: That should be triggered from the kuser postUpdate and not from the service, a user id or screen name could be changed from PS2, internal scripts or maintenance scripts.

**Pull request merged**